### ISSUE TITLE
high IOU in validation but bad results in test

### ISSUE THREAD

**Issue by mirko0x5f** [thanhhau097/chargrid2d#13]
Hello @thanhhau097 ,
I tried to train your implementation with custom data, and I'm getting what I consider high values of IOU (80-90%) in validation, but when I try to run the tests I rarely even get a single result (accuracy < 1% on 4 different labels).
Do you know what could be causing this?
Thank you for open sourcing this.
Best,
Mirko

**Comment by thanhhau097** [comment]
Have you loaded correct checkpoint? If yes, could you please test with validation/train images?

**Comment by mirko0x5f** [comment]
Yes, I loaded the last training checkpoint. I tried with validation images and the result is similar. Very few gets predicted.
Thank you for the fast answer!

**Comment by thanhhau097** [comment]
I thought there may be some problems with the prediction function. If it's correct, the result of validation images should be good. Good luck!
